locales/en-US/app.ftl

```diff
@@ -188,7 +189,7 @@ Home--addon-button-instructions =
 Home--record-instructions =
     To start profiling, click on the profiling button, or use the
     keyboard shortcuts. The icon is blue when a profile is recording.
-    Hit <kbd>Capture Profile</kbd> to load the data into profiler.firefox.com.
```

Comment:
The string is actually just "Capture" in [Firefox](https://searchfox.org/mozilla-central/rev/79d73b4aff88dd4a0f06dd3789e1148c49b0de60/browser/locales/en-US/browser/appmenu.ftl#130)